### ISSUE TITLE
[Question]: uie微调的时候不进入训练迭代

### ISSUE THREAD

**Issue by cloudsurging** [PaddlePaddle/PaddleNLP#5081]
### 请提出你的问题

uie微调的时候不进入训练迭代，显示开始训练
[2023-03-03 11:34:28,598] [    INFO] - ***** Running training *****
[2023-03-03 11:34:28,598] [    INFO] -   Num examples = 200
[2023-03-03 11:34:28,598] [    INFO] -   Num Epochs = 20
[2023-03-03 11:34:28,599] [    INFO] -   Instantaneous batch size per device = 16
[2023-03-03 11:34:28,599] [    INFO] -   Total train batch size (w. parallel, distributed & accumulation) = 16
[2023-03-03 11:34:28,599] [    INFO] -   Gradient Accumulation steps = 1
[2023-03-03 11:34:28,600] [    INFO] -   Total optimization steps = 260.0
[2023-03-03 11:34:28,600] [    INFO] -   Total num train samples = 4000.0
[2023-03-03 11:34:28,613] [    INFO] -   Number of trainable parameters = 117946370

然后就退出了，不进入训练

**Comment by linjieccc** [comment]
@cloudsurging Hi，可以尝试把`per_device_train_batch_size`和`per_device_eval_batch_size`调小，看是否是显存不足的问题

**Comment by cloudsurging** [comment]
batch size 改成8，又行了
